src/assets/scripts/client/InputController.js

```diff
@@ -713,6 +714,16 @@ export default class InputController {
     }
 
     /**
+     * Disable default behavior of right-click menu appearing on right click
+     *
+     * @for InputController
+     * @method _disableRightClickMenu
+     */
+    _disableRightClickMenu() {
```

Comment:
do we see this behavior with 
```javascript
event.preventDefault();
event.stopPropagation();
``` 
or
```javascript
event.preventDefault();
event.stopImmediatePropagation();
```

It just seems odd that we would have to drill into the document this way.